src/main/resources/config_file_documentation.txt

```diff
@@ -87,7 +87,7 @@ Config file must be valid JSON format. Examples can be found under the test fold
         if not supplied: none
 
   "candidates" required
-    list of registered candidate names and associated candidate code
+    list of registered candidate names and associated candidate code (note: leave empty when CVR is in Common Data Format)
```

Comment:
Probably get rid of the * in "Name *" in `GuiConfigLayout.fxml` in light of this. Though at this point I'm feeling like these notes are getting pretty watered down. Maybe we should consider a new symbol (**?) for "required unless CVR is JSON"?

Another thing: you may need to modify the config validation to allow for no candidates if JSON.

Reply:
Well, "Name" is still required if you're actually going to add a candidate. We just don't want you to add any if the input is CDF. I'll enforce that in a separate PR.

> Another thing: you may need to modify the config validation to allow for no candidates if JSON.

This part already works.